tests-hivemq-platform-operator/src/integrationTest/java/com/hivemq/helmcharts/single/HelmCustomServiceAccountIT.java

```diff
@@ -146,4 +189,13 @@ private void createRoleBinding() {
                 .build()).create();
         assertThat(roleBinding).isNotNull();
     }
+
+    private void assertPlatformPodAnnotations() {
+        await().untilAsserted(() -> client.pods()
+                .inNamespace(platformNamespace)
+                .list()
+                .getItems()
+                .forEach(pod -> assertThat(pod.getMetadata().getAnnotations()) //
+                        .containsKey("hivemq/platform-operator-init-app-version")));
```

Comment:
Yeah, the init app always starts a `PodReconciler` that sets this annotation (and optionally `hivemq/platform-errors`).

Not sure how the improve the method name though, since `assertPlatformPodAnnotations()` is exactly what the method does. Are you missing the `why` here? Could maybe solve this with a comment on the method.

Reply:
Comment added.

Reply:
yeah, thanks!